### ISSUE TITLE
Some pages not responsive on small screen

### ISSUE THREAD

**Issue by NatashaRichter** [CenterForOpenScience/cos.io#484]
## Description

When I make the screen smaller, the following pages are not fully responsive, leading to text along the left edge being cut off:
- News
- COS Communities
- COS Board
## Screenshots

**News**

![news_page_cut_off](https://cloud.githubusercontent.com/assets/7093086/9466601/0241d556-4b04-11e5-83ba-d4a2534de99a.png)

**COS Communities**   

![cos_communities_page_cut_off](https://cloud.githubusercontent.com/assets/7093086/9466607/1135b988-4b04-11e5-8d3f-a2b809170c87.png)

**COS Board**

![cos_board_page_cut_off](https://cloud.githubusercontent.com/assets/7093086/9466615/1bca2bfe-4b04-11e5-9754-b159f8f279cf.png)

**Comment by GaryKriebel** [comment]
@caneruguz ... might this be a quick fix?

**Comment by caneruguz** [comment]
Yes I think so. It looks like a grid layout problem. I can take care of this today. 

—

**Comment by saradbowman** [comment]
Thank you!

**Comment by GaryKriebel** [comment]
Thanks
